projects/core/src/cart/bundle/public_api.ts

```diff
@@ -0,0 +1,4 @@
+/*
+ * Public API Surface of bundle
+ */
+export * from './index';
```

Comment:
As discussed offline:
Here we're adding the bundle feature to the `projects/core/src/cart`. We should rather implement any new features in their desired place - in your case the `feature-libs/cart`. AFAIK the saved-carts feature will be merged soon inside the brand new folder `feature-libs/cart` which is a foundation for `@spartacus/cart` lib. And /bundle feature should be moved there, when it's in place.

Just to be clear: the generic logic of the entry-groups should stay in the `projects/core/src/core/...`, where you've implemented it